src/routes/code-of-conduct/+page.svelte

```diff
@@ -11,46 +11,67 @@
 			the following code of conduct.
 		</p>
 		<p>
-			Our conference is dedicated to providing a harassment-free conference experience for everyone,
-			regardless of gender, age, sexual orientation, disability, physical appearance, body size,
-			race, or religion (or lack thereof). We do not tolerate harassment of conference participants
-			in any form. Sexual language and imagery is not appropriate for any conference venue,
-			including talks, workshops, parties, Twitter and other online media. Conference participants
-			violating these rules may be sanctioned or expelled from the conference at the discretion of
-			the conference organisers.
+			DDD South West is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks. Conference participants violating these rules may be sanctioned or expelled from the conference at the discretion of the conference organizers.
 		</p>
 		<p>
-			Harassment includes offensive verbal comments related to gender, age, sexual orientation,
-			disability, physical appearance, body size, race, religion, sexual images in public spaces,
-			deliberate intimidation, stalking, following, harassing photography or recording, sustained
-			disruption of talks or other events, inappropriate physical contact, and unwelcome sexual
-			attention. Participants asked to stop any harassing behavior are expected to comply
-			immediately.
+			Harassment includes, but is not limited to:
 		</p>
+		<ul>
+			<li>Verbal comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, or religion.</li>
```

Comment:
Given the current situation in Gaza I think we should have another point that explicitly states our position on things that could be perceived as antisemitic as most definitions of antisemitism are tied to the existence of the state of Israel. One for discussion?